src/tiles/Tile.tsx

```diff
@@ -141,6 +143,34 @@ export default class Tile<
     }
   };
 
+  handleOnTouch = async () => {
+    if (!this.onTouch) return;
+    Sentry.withScope(async (scope) => {
```

Comment:
this isnt actually overriding transaction names for these events, which means we end up with 'panelkit.boot' for all transactions